subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -1772,15 +1773,6 @@ void ull_cp_rx(struct ll_conn *conn, struct node_rx_pdu *rx)
 
 				/* Process PDU in local procedure */
 				llcp_lr_rx(conn, ctx_l, rx);
-			} else {
-				/* Local active procedure
-				 * Expected local procedure PDU
-				 * Remote active procedure
-				 * Expected remote procedure PDU
-				 */
-
-				/* This cannot happen */
-				LL_ASSERT(0);
 			}
```

Comment:
Do have a dummy `else` clause and include code comment explaining why the `else` condition can be ignored.